mlflow/metrics/utils/genai_metric_definitions.py

```diff
@@ -0,0 +1,36 @@
+from mlflow.exceptions import MlflowException
+from mlflow.metrics.utils.make_genai_metric import make_genai_metric
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, INVALID_PARAMETER_VALUE
+from mlflow.utils.class_utils import _get_class_from_string
+
+DEFAULT_MODEL = "openai:/gpt4"
+LATEST_VERSION = "v1"
+
+
+def correctness(model=DEFAULT_MODEL, version=LATEST_VERSION, examples=[]):
```

Comment:
should we be more explicit that this is the `metric_version` rather than a `model_version`

Reply:
Is the docstring sufficient or do we want to rename this? cc @sunishsheth2009

Reply:
I agree with @prithvikannan suggestion. Lets rename to `metric_version`. I think thats more meaningful.